sale_order_net_price_control/models/res_config_settings.py

```diff
@@ -0,0 +1,10 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    lowest_pricelist_id = fields.Many2one(related="company_id.lowest_pricelist_id", readonly=False, string='MAYM pricelist')
```

Comment:
Remove "MAYM" here. Customer calls it "MAYM" because he will create a pricelist with that name and assign it here, but this name have no sense for us, if we're thinking it as a generic development.